public/controllers/management/components/management/ruleset/views/rule-info.tsx

```diff
@@ -543,11 +583,14 @@ class WzRuleInfo extends Component {
    * @param {Number} ruleId
    */
   changeBetweenRules(ruleId) {
+    // Prevent reloading the same rule
+    if (this.state.currentRuleId == ruleId) return;
```

Comment:
The use of braces is always required, according to our [Styleguide](https://github.com/wazuh/wazuh-kibana-app/blob/master/STYLEGUIDE.md#always-use-braces-for-conditionals-and-loops).